tensorflow/lite/micro/tools/ci_build/test_generate_micro_mutable_op_resolver_tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -e
```

Comment:
Where do you envision to run this test in presubmit?

Reply:
I mostly added this because of a comment from @advaitjain so I don't have a strong opinion but maybe it is enough to run it on a nightly basis?

Reply:
And even if not run regularly it might be useful to test out a new model.